tutorial/under-the-hood/Compilation.md

```diff
@@ -25,14 +25,17 @@ Show the structure of a compiled test suite, both in XSLT and XQuery.
 ### Test suite
```

Comment:
Do you think hyperlinking is useful here (and other places)?
```suggestion
### [Test suite](compilation-simple-suite.xspec)
```

Reply:
I don't see great value in linking to a file that contains the same content. It might even be distracting. I think of the non-MarkDown files in this folder as being mostly for our benefit in maintaining the wiki, not so much for readers' benefit. But if you feel strongly about hyperlinks, I'll commit the one you added and also add some more.

Reply:
Right. Let's not link it.